packages/view/backend/sirius-components-view/src/main/resources/model/view.genmodel

```diff
@@ -5,7 +5,7 @@
     modelDirectory="/sirius-components-view/src/main/java" creationIcons="false" editDirectory="/sirius-components-view-edit/src/main/java"
     editorDirectory="" modelPluginID="sirius-components-view" modelName="View" rootExtendsClass="org.eclipse.emf.ecore.impl.MinimalEObjectImpl$Container"
     codeFormatting="true" commentFormatting="true" importerID="org.eclipse.emf.importer.ecore"
-    bundleManifest="false" complianceLevel="17.0" copyrightFields="false" editPluginID="sirius-components-view-edit"
+    bundleManifest="false" copyrightFields="false" editPluginID="sirius-components-view-edit"
```

Comment:
Why remove the compliance to Java 17?

Reply:
I had an issue with loading the .genmodel it crashed with this property set to 17.0

Reply:
I change the version of `org.eclipse.emf.codegen.ecore` to 2.25.0 in `sirius-components-view-builder`